src/test/java/org/fogbowcloud/manager/core/models/linkedlists/SynchronizedDoublyLinkedListTest.java

```diff
@@ -24,99 +24,133 @@ public void initialize() {
 
     @Test
     public void testAddFirst() {
```

Comment:
The test cases must be described before testing based on the new template.

Reply:
Right! I will fix this in the next commit.